projects/ngrx.io/content/guide/store/reducers.md

```diff
@@ -205,4 +206,4 @@ Whether your feature states are loaded eagerly or lazily depends on the needs of
 
 Reducers are only responsible for deciding which state transitions need to occur for a given action.
 
-[Effects](guide/effects)
+In an application there is also a need to handle impure actions, e.g. AJAX requests, in NgRx we call them [Effects](guide/effects).
```

Comment:
I've also added this because I found that only having `Effects` here is a bit weird.